cmd/ebrelayer/main.go

```diff
@@ -92,21 +92,21 @@ func buildRootCmd() *cobra.Command {
 		initRelayerCmd(),
 		generateBindingsCmd(),
 		replayEthereumCmd(),
-		replayCosmosCmd(),
+		replayCosmosBurnLockCmd(),
+		replayCosmosSignatureAggregationCmd(),
 		listMissedCosmosEventCmd(),
 	)
-
 	return rootCmd
 }
 
 //	initRelayerCmd
 func initRelayerCmd() *cobra.Command {
 	//nolint:lll
 	initRelayerCmd := &cobra.Command{
-		Use:     "init [networkDescriptor] [tendermintNode] [web3Provider] [bridgeRegistryContractAddress] [validatorMnemonic]",
+		Use:     "init [networkDescriptor] [tendermintNode] [web3Provider] [bridgeRegistryContractAddress] [validatorMnemonic] [signatureAggregator]",
 		Short:   "Validate credentials and initialize subscriptions to both chains",
-		Args:    cobra.ExactArgs(5),
-		Example: "ebrelayer init 1 tcp://localhost:26657 ws://localhost:7545/ 0x30753E4A8aad7F8597332E813735Def5dD395028 mnemonic --chain-id=peggy",
+		Args:    cobra.ExactArgs(6),
```

Comment:
Is there any reason to leave signatureAggregation as a required argument over a flag? Shouldn't the system default to 'true' unless a flag says otherwise?

Reply:
We just have one or two dedicated relayers to hangle the signature aggregation events. For not signature aggregation relayers, they just handle the burn/lock messages from sifnode, then they sign against the prophecy id, send the message back to Sifchain. After enought relayers sign for the same prophecy, the signature aggregation completed event emitted. then signature aggregation relayers get all signatures and other info, send a transaction to smart contract in Ethereum.